sdk/datafactory/azure-mgmt-datafactory/azure/mgmt/datafactory/models/tabular_translator.py

```diff
@@ -1,51 +0,0 @@
-# coding=utf-8
```

Comment:
This file gets removed due to Autorest bug - it is treated as not used, but in fact our customers need it. Please do not remove it.

Reply:
ok, I will check this.